src/test/Makefile-client.am

```diff
@@ -719,6 +726,15 @@ ceph_test_rgw_obj_LDADD = \
 ceph_test_rgw_obj_CXXFLAGS = $(UNITTEST_CXXFLAGS)
 bin_DEBUGPROGRAMS += ceph_test_rgw_obj
 
+ceph_test_rgw_bencode_SOURCES = test/rgw/test_rgw_bencode.cc
+ceph_test_rgw_bencode_LDADD = \ 
+        $(LIBRADOS) $(LIBRGW) $(LIBRGW_DEPS) $(CEPH_GLOBAL) \
+        $(UNITTEST_LDADD) $(CRYPTO_LIBS) \
+        -lcurl -luuid -lexpat
+
+ceph_test_rgw_bencode_CXXFLAGS = $(UNITTEST_CXXFLAGS)
+bin_DEBUGPROGRAMS += ceph_test_rgw_bencode
```

Comment:
note that you've added this block to Makefile-client.am twice

and `bin_DEBUGPROGRAMS` should be `check_TESTPROGRAMS` if you want it to run with make check

the naming convention for these is `unittest_` instead of `ceph_test_`